lib/cocoapods/installer/analyzer.rb

```diff
@@ -658,7 +658,22 @@ def generate_pod_target(target_definitions, target_inspections, specs, scope_suf
         platform = determine_platform(specs, target_definitions, host_requires_frameworks)
         file_accessors = create_file_accessors(specs, platform)
         PodTarget.new(sandbox, host_requires_frameworks, user_build_configurations, archs, platform, specs,
-                      target_definitions, file_accessors, scope_suffix)
+                      target_definitions, file_accessors, scope_suffix).tap do |target|
```

Comment:
this looks like the wrong place to be doing this, it should be done in an installer

Reply:
I don't understand